packages/graphql-api/src/resources/event/eventSearch.type.js

```diff
@@ -0,0 +1,257 @@
+const { gql } = require('apollo-server');
```

Comment:
Its not clear why we have moved the GQL type definitions to this file from event.type.js